ui/src/components/connect/OurMinistries.js

```diff
@@ -0,0 +1,145 @@
+            h={[180, 250]}
+            p={12}
+            borderWidth="1px"
+            borderRadius="20"
+            bgImage={`linear-gradient(180deg, rgba(198, 246, 213, 0.7) 0%, rgba(39, 103, 73, 0.7) 100%), url('${process.env.PUBLIC_URL}/images/connect/focus-ministry.jpg')`}
+            bgPosition="center"
+            bgSize="cover"
+            shadow="lg"
+            color="white"
+            className="flipper"
+          >
+            <Heading
+              as="h4"
+              fontSize={["1.5em", "1.5em", "2em"]}
+              fontWeight={700}
+              textAlign="center"
+              className="front"
+            >
+              SINGLE ADULTS
+            </Heading>
+            <Box className="back" fontWeight={600}>
+              Creating an avenue for single working adults to fellowship and
+              learn how to best exercise their God-given influence in their
+              respective schools and workplaces.
+            </Box>
+          </Center>
+        </Box>
+        <Box className="flip-container">
+          <Center
+            h={[180, 250]}
+            p={12}
+            borderWidth="1px"
+            borderRadius="20"
+            bgImage={`linear-gradient(180deg, rgba(252, 129, 129, 0.7) 0%, rgba(151, 38, 109, 0.7) 100%), url('${process.env.PUBLIC_URL}/images/connect/married-couples-ministry.jpg')`}
+            bgPosition="center"
+            bgSize="cover"
+            shadow="lg"
+            color="white"
+            className="flipper"
+          >
+            <Heading
+              as="h4"
+              fontSize={["1.5em", "1.5em", "2em"]}
+              fontWeight={700}
+              textAlign="center"
+              className="front"
+            >
+              MARRIED COUPLES
+            </Heading>
+            <Box className="back" fontWeight={600}>
+              Bringing together Godly families that will love on and impact the
+              next generation of their families and city.
+            </Box>
+          </Center>
+        </Box>
+        <Box className="flip-container">
+          <Center
+            h={[180, 250]}
+            p={12}
+            borderWidth="1px"
+            borderRadius="20"
+            bgImage={`linear-gradient(180deg, rgba(254, 252, 191, 0.7) 0%, rgba(246, 173, 85, 0.7) 100%), url('${process.env.PUBLIC_URL}/images/connect/children-ministry.jpg')`}
+            bgPosition="center"
+            bgSize="cover"
+            shadow="lg"
+            color="white"
+            className="flipper"
+          >
+            <Heading
+              as="h4"
+              fontSize={["1.5em", "1.5em", "2em"]}
+              fontWeight={700}
+              textAlign="center"
+              className="front"
+            >
+              CHILDREN
+            </Heading>
+            <Box className="back" fontWeight={600}>
+              Building Blocks is HMCC’s Children Ministry, learn more about it{" "}
+              <Text as="u">
```

Comment:
arent the links by default underlined?

Reply:
they're default underlined only on hover